test/list-command.test.js

```diff
@@ -1,28 +1,101 @@
 const path = require('path')
 const spawn = require('cross-spawn')
 
+const list = (options = []) => {
+    const result = spawn.sync(
+        path.join(__dirname, '..', 'bin', 'd2-utils-codemods'),
+        ['list', ...options],
+        {
+            cwd: __dirname,
+            env: {
+                ...process.env,
+                RESOLVE_PATH: path.join(__dirname, 'node_modules'),
+            },
+        }
+    )
+
+    if (result.stderr.toString()) {
+        console.log('STDOUT:')
+        console.log(result.stdout.toString())
+        console.log('STDERR:')
+        console.log(result.stderr.toString())
```

Comment:
Probably this should also be a `snapshot` or `inlineSnapshot`?

Reply:
I guess we're actuually doing that later, but it seems incorrect too console.log stderr here?

Reply:
Love the idea! Updated